trace-resources/trace-resources.cabal

```diff
@@ -0,0 +1,76 @@
+cabal-version:        2.4
+name:                 trace-resources
+version:              0.1.0.0
+
+synopsis:             Package for tracing resources for linux, mac and windows
+author:               Juergen Nicklisch-Franken
+maintainer:           operations@iohk.io
+copyright:            2021 IOHK
+extra-source-files:   CHANGELOG.md
+                      README.md
+                      src/Cardano/Logging/Resources/os-support-darwin.h
+                      src/Cardano/Logging/Resources/os-support-win.h
```

Comment:
It's more common to put these files in a top level directory `./include` and add
```
include-dirs: include
```

Reply:
See above

Reply:
> It's more common to put these files in a top level directory `./include` and add

I searched in repos: cardano-node, cardano-base, ouroboros-network and cardano-ledger-specs, and could not find one example of this practice. So it is surely common in C programming, it doesn't seem common practice in this project. Do I need to do this changes?

Reply:
Done the changes